src/test/a11y/a11y.ts

```diff
@@ -20,19 +27,29 @@ interface PallyIssue {
   typeCode: number;
 }
 
+function loginPally(): Pa11yResult {
+  return pa11y(`${config.TEST_URL}/login`, {
+    hideElements: '.govuk-footer__licence-logo, .govuk-header__logotype-crown',
+    actions: [
+      `set field #username to ${config.TestUser}`,
+      `set field #password to ${config.TestPass}`,
+      'click element .button',
+      'wait for path to be /',
+    ],
+  });
```

Comment:
Looks like each call of `pa11y()` spins up it's own browser so I don't think you can reuse instances 😞